terraform/environments/delius-core/platform_base_variables.tf

```diff
@@ -9,3 +9,8 @@ variable "collaborator_access" {
   default     = "developer"
   description = "Collaborators must specify which access level they are using, eg set an environment variable of export TF_VAR_collaborator_access=migration"
 }
+
```

Comment:
We shouldn't edit this file, it's created by the mod platform application creation process.

The environment already exists as `local.environment`.

Reply:
Removed all changes to that file